tests/foreman/cli/test_repository.py

```diff
@@ -361,8 +523,15 @@ def test_positive_update_url(self):
         @Assert: Repository url is updated
         """
         new_repo = self._make_repository()
+        # generate a tuple of auth repos with all valid credentials
+        auth_repos = ()
```

Comment:
yes, I do realize. I am looking from QA standpoint only here, not from logic perspective :)